src/index.ts

```diff
@@ -21,9 +21,9 @@ export = class OutputCsv {
   public async serve(req: Request, res: Response) {
     res.setHeader('Content-Type', 'text/csv');
     try {
-      const csvTemplate = _.get(req, 'res.locals.csvTemplate') as Record<string, any>;
-      const csvTemplateTransforms = _.get(req, 'app.locals.csvTemplateTransforms') as TransformsList;
-      const csvFileName = _.get(req, 'app.locals.csvFileName', 'output');
+      const csvTemplate = _.get(req, 'res.locals.csvTemplate') || _.get(req, 'app.locals.csvTemplate') as Record<string, any>;
```

Comment:
Typescript question: Can you reliably cast these as `Record<string, any>` since the result of `_.get` might be `undefined`?  Also, I think you don't need `_.get` here. because `req.res.locals` and `req.app.locals` should always be defined on the express req object.

Reply:
That is a good question! We can't reliably cast without some sort of type-guarding or type-narrowing. I have implemented type-guarding for `csvTemplate` for that. I've removed `_.get`  and directly used `req.res.locals` and `req.app.locals` instead.